fuzz/fuzzers/server.rs

```diff
@@ -6,9 +6,47 @@ extern crate rustls;
 use std::io;
 use std::sync::Arc;
 
-use rustls::server::ResolvesServerCert;
+use rustls::server::{Acceptor, ResolvesServerCert};
 use rustls::{ServerConfig, ServerConnection};
 
+fuzz_target!(|data: &[u8]| {
+    match data.split_first() {
```

Comment:
would this be more effective dispatching on the MSB instead?

Reply:
This comment falls away with my understanding from [the other thread](https://github.com/rustls/rustls/pull/2249#discussion_r1869969109).